tf_existing_net_rds_ec_ec2/main.tf

```diff
@@ -12,8 +12,8 @@ terraform {
 # this is for an aws specific provider(not gcp or azure)
 provider "aws" {
   version = "~> 2.44.0"
-  region  = var.region
-  profile = var.profile
+  region  = "us-east-1"
+  profile = "aws-cloudarch-sandbox:aws-cloudarch-sandbox-admin"
```

Comment:
@smithatlanta Any reason why this is hard-coded into the terraform script?

Reply:
Thanks. Just fixed that.